docs/index.rst

```diff
@@ -42,71 +42,64 @@ Contents
    :caption: Introduction
    :titlesonly:
 
-   introduction
+   introduction/about-open-ms.md
+   introduction/background.md
+   introduction/how-openms-works.md
 
 .. toctree::
    :maxdepth: 2
    :caption: Getting Started
```

Comment:
Since we moved the build from source to the developer section, we could rename "Getting Started" to "Installation" and remove the redundancy from the items ("Installation on Windows" -- > "Windows").

Reply:
Done.